src/event.js

```diff
@@ -11,12 +12,24 @@ export const customEvents = popup => {
 		onExpand = () => {
 			popup.element.component.wikipediapreviews.classList.add( 'expanded' )
 		},
+
+		onTouchStart = e => {
+			const targetClassName = e.target.parentNode.className
+			if ( targetClassName.indexOf( 'wikipediapreviews-' ) === -1 ) {
+				popup.hide()
+			}
```

Comment:
one case found, in popup mode, touch the text in `bold`, for example cat article, touch `cat` `domestic cat` `house cat` `farm cat` `feral cat` will close the popup, but it shouldn't.

Reply:
Good catch 👀

Reply:
use the similar code in `onMouseLeave` event should work to cover all elements in the popup

```js
!popup.element.contains( toElement )
```

Reply:
Done